mesonbuild/modules/windows.py

```diff
@@ -206,6 +206,7 @@ def get_names() -> T.Iterable[T.Tuple[str, str, T.Union[str, mesonlib.File, buil
                 depfile=depfile,
                 depend_files=wrc_depend_files,
                 extra_depends=wrc_depends,
+                description='Generating Windows resource {}',
```

Comment:
```suggestion
                description='Compiling Windows resource {}',
```

This is actually a compiler.